server/src/test/java/org/candlepin/dto/api/v1/PermissionBlueprintInfoTranslatorTest.java

```diff
@@ -78,7 +78,14 @@ protected void verifyOutput(PermissionBlueprintInfo source, PermissionBlueprintD
             assertEquals(source.getAccessLevel(), dest.getAccess());
 
             if (childrenGenerated) {
-                this.ownerTranslatorTest.verifyOutput(source.getOwner(), dest.getOwner(), true);
+
+                OwnerDTO ownerDTO = new OwnerDTO();
+                ownerDTO.setId(dest.getOwner().getId());
+                ownerDTO.setKey(dest.getOwner().getKey());
+                ownerDTO.setDisplayName(dest.getOwner().getDisplayName());
+
+                this.ownerTranslatorTest.verifyOutput(source.getOwner(),
+                    ownerDTO, true);
```

Comment:
We shouldn'e be creating any DTOs (or entities) in a translation verification test. In fact, since we moved to a manual translation of NestedOWner in `PermissionBlueprintInfoTranslator`, the verification should happen outside of the `if (childrenGenerated)` block, by a series of `assertEquals` calls (one for each field), and the ownerTranslatorTest field should be removed (as it is no longer needed).

Reply:
Corrected it.